kubernetes/samples/scripts/delete-weblogic-domain-resources.sh

```diff
@@ -73,18 +73,14 @@ EOF
 #
 function getDomainResources {
   local domain_regex=''
-  if [ "$1" = "all" ]; then
-    LABEL_SELECTOR="weblogic.domainUID"
-  else
-    LABEL_SELECTOR="weblogic.domainUID in ($1)"
-    IFS=',' read -ra UIDS <<< "$1"
-    for i in "${!UIDS[@]}"; do
-      if [ $i -gt 0 ]; then
-        domain_regex="$domain_regex|"
-      fi
-      domain_regex="$domain_regex^Domain ${UIDS[$i]} "
-    done
-  fi
+  LABEL_SELECTOR="weblogic.domainUID in ($1)"
+  IFS=',' read -ra UIDS <<< "$1"
+  for i in "${!UIDS[@]}"; do
+    if [ $i -gt 0 ]; then
+      domain_regex="$domain_regex|"
+    fi
+    domain_regex="$domain_regex^Domain ${UIDS[$i]} "
```

Comment:
I believe that Dongbo has updated (or is updating) the sample create domain scripts to add the weblogic.domainUID label to the domain resource too, so I don't think the delete script needs to have a special rule to remove the domain resource.

Reply:
Ok. @doxiao, are your changes in develop already?

Reply:
Not yet. See https://github.com/oracle/weblogic-kubernetes-operator/pull/552.